packages/web/src/scss/foundation/_links.scss

```diff
@@ -7,11 +7,15 @@
 a {
     color: tokens.$action-link-primary-default;
 
-    &:hover {
-        color: tokens.$action-link-primary-hover;
-    }
-
     &:active {
```

Comment:
![image](https://user-images.githubusercontent.com/7041807/220347664-34d3aea9-2d66-43cb-898e-206e020b4d72.png)
`packages/web/src/scss/helpers/_links.scss` please unify how the underline looks.